proto/hummock.proto

```diff
@@ -328,6 +328,7 @@ message CompactTask {
     JOIN_HANDLE_FAILED = 11;
     TRACK_SST_OBJECT_ID_FAILED = 12;
     NO_AVAIL_CPU_RESOURCE_CANCELED = 13;
+    HEARTBEAT_PROGRESS_CANCELED = 14;
```

Comment:
Is it unused ?

Reply:
Divide HeartbeatCancel into two types for better debugging